modules/private-clusters-setting-dns-private.adoc

```diff
@@ -6,11 +6,17 @@
 [id="private-clusters-setting-dns-private_{context}"]
 = Setting DNS to private
 
-After you deploy a cluster, you can modify its DNS to use only a private zone.
+After you deploy a cluster, you can modify its DNS to use only a private zone, by modifying the `DNS` custom resource (CR).
+Because the Ingress Controller consults the `DNS` CR definition when it creates `Ingress` objects, when you create or modify `Ingress` objects, only private records are created. This secures the cluster by preventing public DNS records from being created.
```

Comment:
I don;t think this line should be moved here, because that is taking it out of context.  It will only do this after you apply the patch this is listed below, where this was moved from.

I would not use the phrase "secures the cluster" because that is a guarantee we cannot make.

Reply:
ACK, moved back and removed "secures the cluster"